moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
+                            "minItems": 1,
+                            "items": {
+                              "title": "License",
+                              "type": "object",
+                              "description": "This schema describes the attributes of a license.\n",
+                              "properties": {
+                                "identifier": {
+                                  "description": "A license according to https://spdx.org/licenses/ or \"proprietary\".",
+                                  "example": "https://creativecommons.org/publicdomain/zero/1.0",
+                                  "format": "iri",
+                                  "oneOf": [
+                                    {
+                                      "format": "iri",
+                                      "example": "https://creativecommons.org/publicdomain/zero/1.0",
+                                      "description": " A generic, general license according to https://spdx.org/licenses/."
+                                    },
+                                    {
+                                      "enum": [
+                                        "proprietary"
+                                      ],
+                                      "example": "proprietary",
+                                      "description": "Marking the course as proprietary."
+                                    }
+                                  ],
+                                  "type": "string"
+                                },
+                                "url": {
+                                  "type": [
+                                    "string",
+                                    "null"
+                                  ],
+                                  "format": "iri",
+                                  "example": null,
+                                  "description": "A link to the specific license document (if any, otherwise set to \"null\" - the identifier pointing at the general license is to be considered the license document then). Has to be \"null\" if identifier is \"proprietary\"."
+                                }
+                              },
+                              "required": [
+                                "identifier"
+                              ]
+                            }
+                          }
+                        },
+                        "required": [
+                          "contentUrl",
+                          "license"
+                        ]
+                      }
+                    },
+                    "required": [
+                      "name",
+                      "type"
+                    ]
+                  }
+                },
+                "required": [
+                  "name",
+                  "type"
+                ]
+              }
+            ]
+          }
+        },
+        "teaches": {
```

Comment:
Similar to above, active wording vs. nouns.

Reply:
This is also vocabulary which is given by schema.org.